code-of-conduct.md

```diff
@@ -0,0 +1,46 @@
+
+Participants are responsible for knowing and abiding by these rules.
+
+**Community Guidelines:**
+
+* **Be friendly and patient**: Remember you might not be communicating in someone else's primary spoken or programming language, and others may not have your level of understanding.
+* **Be welcoming**: Our communities welcome and support people of all backgrounds and identities. This includes, but is not limited to members of any race, ethnicity, culture, national origin, color, immigration status, social and economic class, educational level, sex, sexual orientation, gender identity and expression, age, size, family status, political belief, religion, and mental and physical ability.
+Harassment includes:
+* **Be respectful:** We are a world-wide community of professionals, and we conduct ourselves professionally. Disagreement is no excuse for poor behavior and poor manners. 
+
+**Disrespectful and unacceptable behavior includes, but is not limited to:**
+* Violent threats or language.
+* Discriminatory or derogatory jokes and language.
+* Posting sexually explicit or violent material.
+* Posting, or threatening to post, people's personally identifying information ("doxing").
+* Insults, especially those using discriminatory terms or slurs.
+* Behavior that could be perceived as sexual attention.
+* Advocating for or encouraging any of the above behaviors.
+* Offensive comments related to gender, gender identity and expression, sexual orientation, disability, mental illness, neuro(a)typicality, physical appearance, body size, age, race, or religion.
+* Unwelcome comments regarding a person's lifestyle choices and practices, including those related to food, health, parenting, drugs, and employment.
+* Deliberate misgendering or use of 'dead' or rejected names.
+* Gratuitous or off-topic sexual images or behaviour in spaces where they're not appropriate.
+* Physical contact and simulated physical contact (eg, textual descriptions like "hug" or "backrub") without consent or after a request to stop.
+* Threats of violence.
+* Incitement of violence towards any individual, including encouraging a person to commit suicide or to engage in self-harm.
+* Deliberate intimidation.
+* Stalking or following.
+* Harassing photography or recording, including logging online activity for harassment purposes.
+* Sustained disruption of discussion.
+* Unwelcome sexual attention.
+* Pattern of inappropriate social contact, such as requesting/assuming inappropriate levels of intimacy with others.
+* Continued one-on-one communication after requests to cease.
+* Deliberate "outing" of any aspect of a person's identity without their consent except as necessary to protect vulnerable people from intentional abuse.
```

Comment:
I think this is covered under "doxing" above - or maybe we should combine them (?)